frontend/lib/api/gen/openapi.ts

```diff
@@ -182,7 +195,7 @@ Bijlage 2: uitkomsten per stembureau" from the
 export interface PollingStationResults {
   differences_counts: DifferencesCounts;
   political_group_votes: PoliticalGroupVotes[];
-  recounted?: boolean;
+  recounted?: boolean | null;
```

Comment:
```suggestion
  recounted?: boolean;
```

Reply:
Because you have to manually re-run the gen command, unfortunately that's not built into commit/push yet